website/docs/pact_broker/advanced_topics/provider_verification_badges.md

```diff
@@ -6,19 +6,19 @@ _See the the_ [_Badges_](/pact_broker/configuration/features#badges) _section of
 
 If you are publishing [Provider verification results](provider_verification_results.md) to your Pact Broker \(v2.3.0+\), you can also display the verification status in your READMEs using a shiny badge like this one:
 
-[![Pact Status](https://test.pactflow.io/pacticipants/FrontendWebsite/branches/test/latest-version/can-i-deploy/to-environment/test/badge)](https://test.pactflow.io/pacticipants/FrontendWebsite/branches/test/latest-version/can-i-deploy/to-environment/test)
+[![Pact Status](https://test.pactflow.io/pacticipants/pactflow-example-consumer/branches/master/latest-version/can-i-deploy/to-environment/prod/badge)](https://test.pactflow.io/pacticipants/pactflow-example-consumer/branches/master/latest-version/can-i-deploy/to-environment/production)
 
 ## Can I Deploy badge
 
 [![Can I Deploy Status](/img/can-i-deploy-badge.svg)](https://test.pactflow.io)
 
 ### Using branches and environments
 
-Requires version 2.94+ of the Pact Broker
+Requires version 2.XYZ of the Pact Broker
```

Comment:
[2.111.0](https://github.com/pact-foundation/pact_broker/releases/tag/v2.111.0) 

I think we should retain this as 2.94+ for can-i-deploy badge, and add 2.111.0 under the can-i-merge badge, to be clear.

Reply:
yeah, this was a copy/pasta error. Resolved.